scripts/lib/libvirt/libvirt_setup.py

```diff
@@ -105,7 +105,7 @@ def compute_config(args, cpu_flags=cpuflags(), machine=None):
         targetdevprefix = "sd"
         targetbus = "ide"
     controller_raid_volumes = args.controller_raid_volumes
-    if args.nodecounter != "1":
+    if args.nodecounter > args.numcontrollers:
```

Comment:
this is my lack of understanding here: not sure about the meaning of `nodecounter` in reference to `numcontrollers`. Do we need to subtract somewhere to know the number of compute nodes? Why comparing both here?